From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: atlochowski <hucul@o2.pl>
Date: Fri, 20 Mar 2020 14:37:01 +0000
Subject: [PATCH] Update for Polish networks. MER#902

Changes to Polish networks:
- added apn mms to netia
- added apn mms to vectra
- removed telogin apn
- changed cyfrowy polsat apn value
- removed apn duplicate from plus gsm
- removed mms proxy from play
- removed mni mms conf
---
 serviceproviders.xml | 49 ++++++++++++--------------------------------
 1 file changed, 13 insertions(+), 36 deletions(-)

diff --git a/serviceproviders.xml b/serviceproviders.xml
index a1c99749f..8697128a3 100644
--- a/serviceproviders.xml
+++ b/serviceproviders.xml
@@ -11635,7 +11635,6 @@ conceived.
 				<usage type="mms"/>
 				<name>Play MMS</name>
 				<mmsc>http://10.10.28.164/mms/wapenc</mmsc>
-				<mmsproxy>10.10.25.5:8080</mmsproxy>
 			</apn>
 		</gsm>
 	</provider>
@@ -11700,14 +11699,6 @@ conceived.
 				<mmsc>http://mms.orange.pl</mmsc>
 				<mmsproxy>192.168.6.104:8080</mmsproxy>
 			</apn>
-			<apn value="mni.mms">
-				<usage type="mms"/>
-				<name>MNI MMS</name>
-				<username>mni.mms</username>
-				<password>mni.mms</password>
-				<mmsc>http://mms.mni.pl/</mmsc>
-				<mmsproxy>192.168.6.135:8080</mmsproxy>
-			</apn>
 		</gsm>
 	</provider>
 	<provider primary="true">
@@ -11750,13 +11741,6 @@ conceived.
 				<username>plusgsm</username>
 				<password>plusgsm</password>
 			</apn>
-			<apn value="optimizer">
-				<plan type="postpaid"/>
-				<plan type="prepaid"/>
-				<usage type="internet"/>
-				<name>iPlus optimizer with data compression</name>
-				<name xml:lang="pl">iPlus optimizer z kompresją danych</name>
-			</apn>
 			<apn value="mms">
 				<usage type="mms"/>
 				<name>Plus MMS</name>
@@ -11769,19 +11753,13 @@ conceived.
 				<mmsc>http://mms.fmgmobile.pl:8002/</mmsc>
 				<mmsproxy>212.2.96.16:8080</mmsproxy>
 			</apn>
-			<apn value="mms.plusgsm.pl">
-				<usage type="mms"/>
-				<name>Plus MMS</name>
-				<mmsc>http://mms.plusgsm.pl:8002</mmsc>
-				<mmsproxy>212.2.96.16:8080</mmsproxy>
-			</apn>
 		</gsm>
 	</provider>
 	<provider primary="true">
 		<name>Cyfrowy Polsat</name>
 		<gsm>
 			<network-id mcc="260" mnc="12"/>
-			<apn value="multi.internet">
+			<apn value="internet.cyfrowypolsat.pl">
 				<plan type="postpaid"/>
 				<usage type="internet"/>
 			</apn>
@@ -11815,18 +11793,6 @@ conceived.
 				<username>internet</username>
 				<password>internet</password>
 			</apn>
-			<apn value="mni.internet">
-				<plan type="postpaid"/>
-				<usage type="internet"/>
-				<name>APN: mni.internet</name>
-				<username>mni.internet</username>
-			</apn>
-			<apn value="telogic.internet">
-				<plan type="postpaid"/>
-				<usage type="internet"/>
-				<name>APN: telogic.internet</name>
-				<username>telogic.internet</username>
-			</apn>
 		</gsm>
 		<cdma>
 			<username>cdma@orange</username>
@@ -11899,6 +11865,12 @@ conceived.
 				<plan type="postpaid"/>
 				<usage type="internet"/>
 			</apn>
+			<apn value="netia.mms">
+				<usage type="mms"/>
+				<name>Netia MMS</name>
+				<mmsc>http://netia.mms:8002</mmsc>
+				<mmsproxy>212.2.96.16:8080</mmsproxy>
+			</apn>
 		</gsm>
 	</provider>
 	<provider>
@@ -11909,13 +11881,18 @@ conceived.
 				<plan type="postpaid"/>
 				<usage type="internet"/>
 			</apn>
+			<apn value="MMS">
+				<usage type="mms"/>
+				<name>Vectra MMS</name>
+				<mmsc>http://mmsc.play.pl/mms/wapenc</mmsc>
+			</apn>
 		</gsm>
 	</provider>
 	<provider>
 		<name>INEA</name>
 		<gsm>
 			<network-id mcc="260" mnc="03"/>
-			<apn value="telogic.internet">
+			<apn value="internet">
 				<plan type="postpaid"/>
 				<usage type="internet"/>
 				<username>internet</username>
